dot/telemetry/telemetry.go

```diff
@@ -65,13 +66,16 @@ func GetInstance() *Handler {
 				}
 				log.SetOutput(&handlerInstance.buf)
 				log.SetFormatter(new(MyJSONFormatter))
+				go handlerInstance.sender()
 			})
 	}
 	return handlerInstance
 }
 
 // AddConnections adds connections to telemetry sever
 func (h *Handler) AddConnections(conns []*genesis.TelemetryEndpoint) {
+	h.Lock()
+	defer h.Unlock()
 	for _, v := range conns {
 		c, _, err := websocket.DefaultDialer.Dial(v.Endpoint, nil)
 		if err != nil {
```

Comment:
We should continue adding connections on error.

Reply:
Removed return.